Installing-Oppia-(Linux;-Python-3).md

```diff
@@ -145,13 +153,15 @@ exec "$SHELL"
    installed python-3.9.20 to /home/user/.pyenv/versions/3.9.20
    ```
 
-5. Install direnv
+5. Install direnv.
```

Comment:
Install direnv, which helps to set and unset the virtual environment based on the current directory: